lib/Test/Mock/Redis.pm

```diff
@@ -463,7 +463,14 @@ sub llen {
 sub lrange {
     my ( $self, $key, $start, $end ) = @_;
 
-    return @{ $self->_stash->{$key} }[$start..$end];
+    my $list = $self->_stash->{$key};
+
+    $start < 0 and $start = @$list + $start;
```

Comment:
I can't say I'm a huge fan of using "and" for flow control like this.  Would you mind writing it using if blocks?